contracts/test/_metastrategies-fixtures.js

```diff
@@ -0,0 +1,221 @@
+const { loadFixture } = require("ethereum-waffle");
+const { ousdUnits } = require("./helpers");
+const { convexMetaVaultFixture } = require("./_fixture");
+
+// NOTE: This can cause a change in setup from mainnet.
+// However, mint/redeem tests, without any changes, are tested
+// in vault.fork-test.js, so this should be fine.
+
+async function withDefaultStrategiesSet() {
+  const fixture = await loadFixture(convexMetaVaultFixture);
+
+  const { vault, governor, usdt, usdc, OUSDmetaStrategy } = fixture;
+
+  await vault
+    .connect(governor)
+    .setAssetDefaultStrategy(usdt.address, OUSDmetaStrategy.address);
+
+  await vault
+    .connect(governor)
+    .setAssetDefaultStrategy(usdc.address, OUSDmetaStrategy.address);
+
+  return fixture;
+}
+
+async function withBalancedMetaPool() {
+  const fixture = await loadFixture(withDefaultStrategiesSet);
+
+  await balanceMetaPool(fixture);
+
+  return fixture;
+}
+
+async function balanceMetaPool(fixture) {
+  const { vault, domen, ousdMetaPool } = fixture;
+
+  // Balance metapool
+  const ousdBalance = await ousdMetaPool.balances(0);
+  const crv3Balance = await ousdMetaPool.balances(1);
+
+  const exchangeSign = "exchange(int128,int128,uint256,uint256)";
+  const exchagneMethod = await ousdMetaPool.connect(domen)[exchangeSign];
+  if (ousdBalance.gt(crv3Balance)) {
+    // Tilt to 3CRV
+    await exchagneMethod(1, 0, ousdBalance.sub(crv3Balance).div(2), 0);
+  } else if (crv3Balance.gt(ousdBalance)) {
+    // Tilt to OUSD
+    await exchagneMethod(0, 1, crv3Balance.sub(ousdBalance).div(2), 0);
+  }
+
+  await vault.connect(domen).allocate();
+  await vault.connect(domen).rebase();
+}
+
+async function withCRV3TitledMetapool() {
+  const fixture = await loadFixture(withBalancedMetaPool);
+
+  await tiltTo3CRV(fixture);
+
+  return fixture;
+}
+
+async function tiltTo3CRV(fixture, amount) {
+  const { vault, domen, ousdMetaPool } = fixture;
+
+  amount = amount || ousdUnits("500000");
+
+  // Tilt to 3CRV by half a million
+  const exchangeSign = "exchange(int128,int128,uint256,uint256)";
+  await ousdMetaPool.connect(domen)[exchangeSign](1, 0, amount, 0);
```

Comment:
I think this will double the tilt? Also the pool should be balanced before the tilt happens. Since if pool has 10 OUSD and 10 3crv and you exchange 5 CRV for OUSD pool will have 15CRV and 5 OUSD. End difference is 10. For that reason I would first: 
- balance pool
- exchange half of the tilt

Reply:
This function (and also `tiltToOUSD()`) is only called by `withLiquidityOn3CRVTitledPool()` or `withCRV3TitledMetapool()`. Both of which balance the pool before doing the tilt.

> End difference is 10
I'm sorry, I'm a little off the loop here. Do you mean to say the difference shouldn't be high or should it be higher than this?

Reply:
Ah, yeah, that makes sense. Will change it